src/components/vm/consoles/vnc.tsx

```diff
@@ -481,6 +516,7 @@ export class VncActive extends React.Component<VncActiveProps, VncActiveState> {
 
     onSecurityFailure(reason: string | undefined) {
         console.info('Security failure:', reason || "unknown reason");
+        this.props.state.setDisconnected(reason || _("Security failure"));
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2038-e5e6d0bb-20250917-080113-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)